prime-angular-frontend/src/app/core/services/logger.service.ts

```diff
@@ -1,12 +1,15 @@
 import { Injectable } from '@angular/core';
+import { LoggerResource } from '@core/resources/logger-resource.service';
 
 import { environment } from '@env/environment';
 
 @Injectable({
   providedIn: 'root'
 })
 export class LoggerService {
```

Comment:
@hogmann Not related to your changes exactly but @mtpultz , can we get rid of the `log` and `trace` methods?  Don't appear to be used and they are confusing.

Reply:
Going to have to resolve.